specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.5-preview.1/backuprestore.json

```diff
@@ -376,6 +352,11 @@
         "token": {
           "type": "string",
           "description": "The SAS token pointing to an Azure Blob storage container"
+        },
+        "useManagedIdentityStorageAuthentication": {
```

Comment:
It's descriptive, but many of the REST API and SDK architects aren't big on long names. Perhaps something like `useManagedIdentity`? "Storage" and "Authentication" both seem extraneous given the context.